ilastik/applets/pixelClassification/opPixelClassification.py

```diff
@@ -599,8 +610,6 @@ def setupOutputs(self):
         self.prediction_cache_gui.BlockShape.setValue( (blockShapeX, blockShapeY, blockShapeZ) )
         self.opUncertaintyCache.BlockShape.setValue( (blockShapeX, blockShapeY, blockShapeZ) )
 
-        assert self.opConvertPMapsToInputPixelType.Output.meta.drange == self.InputImage.meta.drange
```

Comment:
Why should this check be needed? This gave me an error as `self.InputImage.meta.drange` was `None` at the time of  the check.

Reply:
agreed